frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/Toolbar/DesktopToolbar.tsx

```diff
@@ -1,14 +1,18 @@
 import * as ToolbarPrimitive from '@radix-ui/react-toolbar'
 import type { FC } from 'react'
+import styles from './DesktopToolbar.module.css'
 import { FitviewButton } from './FitviewButton'
 import { ShowModeMenu } from './ShowModeMenu'
 import { TidyUpButton } from './TidyUpButton'
-import styles from './Toolbar.module.css'
 import { ZoomControls } from './ZoomControls'
 
-export const Toolbar: FC = () => {
+type Props = {
+  className?: string
+}
+
+export const DesktopToolbar: FC<Props> = ({ className }) => {
   return (
-    <ToolbarPrimitive.Root className={styles.root}>
+    <ToolbarPrimitive.Root className={`${styles.root} ${className || ''}`}>
```

Comment:
[nits]
It looks like `clsx` could be useful here.

```suggestion
    <ToolbarPrimitive.Root className={clsx(styles.root, className)}>
```

Reply:
https://github.com/liam-hq/liam/commit/862c95332be46d6e9d12b27a81294f4097c342bb applied!